src/utils/Symbol_observable.js

```diff
@@ -0,0 +1,18 @@
+let $$observable
+
+if (typeof Symbol === 'function') {
+  if (!Symbol.observable) {
+    if (typeof Symbol.for === 'function') {
+      $$observable = Symbol.for('observable')
+    } else {
+      $$observable = Symbol()
+    }
+    Symbol.observable = $$observable
+  } else {
+    $$observable = Symbol.observable
+  }
+} else {
+  $$observable = '@@observable'
+}
+
+export default $$observable
```

Comment:
In the meantime, this _could_ ship as-is, if blessed. And I'll be happy to come back and make a change to use a separate module. I'll leave that up to @gaearon. 

FWIW, Falcor has similar concerns. But it was determined that it's not a *huge* deal.